src/settings/SettingsModal.tsx

```diff
@@ -95,6 +102,30 @@ const SettingsModal = ({
                 onChange={onChangeTooltipsToggle}
               />
             </FormControl>
+            <RenderIf shouldRender={getBoolFromEnv('VITE_DEBUG_MODE')}>
+              <Text as="b">Scene Navigation</Text>
+              <Grid display="flow">
+                {Object.keys(SceneName)
+                  .filter(
+                    (key) =>
+                      isNaN(Number(SceneName[key as keyof typeof SceneName])) &&
+                      SceneName[key as keyof typeof SceneName].toString() !=
+                        'UNSET'
```

Comment:
Can you just do `SceneName[key as keyof typeof SceneName] != SceneName.UNSET` here?